app/cmd/plugin_search_test.go

```diff
@@ -76,13 +76,71 @@ var _ = Describe("plugin search command", func() {
 			rootCmd.SetOutput(buf)
 			_, err = rootCmd.ExecuteC()
 			Expect(err).To(BeNil())
-			Expect(buf.String()).To(Equal(`number name       installed version   usedVersion title
-0      fake-ocean true      1.19.0... 1.18.1...   fake-ocean
-1      fake-ln    true      1.19.0... 1.18.1      fake-ln
-2      fake-is    true      1.19.1    1.18.1...   fake-is
-3      fake-oa    false     1.13.0...             fake-oa
-4      fake-open  false     1.13.0                fake-open
-5      fake       true                1.0         fake
+			Expect(buf.String()).To(Equal(`number name       installed version  installedVersion title
+0      fake-ocean true      1.19.011 1.18.111         fake-ocean
+1      fake-ln    true      1.19.011 1.18.1           fake-ln
+2      fake-is    true      1.19.1   1.18.111         fake-is
+3      fake-oa    false     1.13.011                  fake-oa
+4      fake-open  false     1.13.0                    fake-open
+5      fake       true               1.0              fake
+`))
+		})
+
+		It("should success, empty updateCenter list", func() {
+			data, err := generateSampleConfig()
+			Expect(err).To(BeNil())
+			err = ioutil.WriteFile(rootOptions.ConfigFile, data, 0664)
+			Expect(err).To(BeNil())
+
+			request, _ := client.PrepareForManyAvaiablePlugin(roundTripper, "http://localhost:8080/jenkins")
+			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
+			request, _ = client.RequestNullUpdateCenter(roundTripper, "http://localhost:8080/jenkins")
+			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
+			request, _ = client.PrepareForManyInstalledPlugin(roundTripper, "http://localhost:8080/jenkins")
+			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
+			rootCmd.SetArgs([]string{"plugin", "search", "fake"})
+
+			buf := new(bytes.Buffer)
+			rootCmd.SetOutput(buf)
+			_, err = rootCmd.ExecuteC()
+			Expect(err).To(BeNil())
+
+			Expect(buf.String()).To(Equal(`number name       installed version installedVersion title
```

Comment:
This behaviour is not matched with the case title which is `empty updateCenter list`. I believe that this case is not an empty update center list.

Reply:
I have tested when the received site is 500 Status code